packages/@okta/vuepress-site/docs/guides/third-party-risk-integration/main/index.md

```diff
@@ -0,0 +1,348 @@
+    "scope": "okta.riskEvents.manage"
+    }
+    ```
+    >**Note**: The access token expires in 60 minutes (or the value set in the property `expires_in`).
+
+For further background information on this process, see [Get an access token](/docs/guides/implement-oauth-for-okta-serviceapp/main/#get-an-access-token).
+
+### Send a risk event to Okta
+
+This procedure sends a sample risk event payload to the Okta org.
+
+1. Call the following POST API from the Risk Integration Postman collection: **Partner: API to send RiskEvents (Auth using access token)** (`/api/v1/risk/events/ip`). A sample payload follows, which includes two events:
+
+    ```JSON
+    [
+     {
+       "timestamp": "2021-01-20T00:00:00.001Z",
+       "subjects": [
+          {
+            "ip": "6.7.6.7",
+            "riskLevel": "LOW",
+            "message": "none"
+          },
+          {
+            "ip": "1.1.1.1",
+            "riskLevel": "HIGH" ,
+            "message": "Detected Attack tooling and suspicious activity"
+          }
+        ]
+     }
+    ]
+    ```
+
+2. Review the request status of the API call. If the status is `202 Accepted`, the risk events were consumed by the Okta org.
+
+> **Note:** Rate limits of three calls per minute, per risk provider, apply to the `/api/v1/risk/events/ip` endpoint. Each call can contain multiple risk events.
+
+For reference information on this API, see [Risk Events](/docs/reference/api/risk-events).
+
+### Confirm the response
+
+This procedure reviews the Admin Console's System Log to identify the risk event information logged after calling the Risk Event API.
+
+1. Sign in to your Okta org as an administrator.
+2. In the Admin Console, go to **Reports** > **System Log**.
+3. Review the log file or search for the event `security.risk.signal.consume`, which is logged when a risk provider sends a risk event to Okta.
+4. With a risk action of `enforce_and_log`, and a risk-based policy setup, the third-party risk provider event is used when calculating the authentication risk. This information is logged in the `user.session.start` event.
+
+### General troubleshooting tips
+* Save the Postman environment after every change to the environment. Confirm that you are using the correct Postman environment.
+* Postman environment variables are case sensitive. Ensure there are no typos, no leading/trailing spaces, no leading or trailing double quotes in the environment variables.
+* The org URL should not be the admin URL (for example, use `https://demo-org.oktapreview.com` instead of  `https://demo-org-admin.oktapreview.com`) and you don't need a trailing `/` at the end of the URL.
```

Comment:
```suggestion
* The org URL shouldn't be the admin URL (for example, use `https://demo-org.oktapreview.com` instead of  `https://demo-org-admin.oktapreview.com`). You don't need a trailing `/` at the end of the URL.
```